editoast/src/client/mod.rs

```diff
@@ -126,7 +126,7 @@ pub struct GenerateArgs {
 #[command(about, long_about = "Clear infra generated data")]
 pub struct ClearArgs {
     /// List of infra ids
-    pub infra_ids: Vec<u64>,
+    pub infra_ids: Vec<i64>,
```

Comment:
Why should we change that? Requiring a positive value seems like a good idea.

Reply:
The base type is i64 and we switch to u64 only for CLI args, we'll need to cast each time.
It's better to keep consistency: either change all infra_id to u64 or keep them all as i64.